src/types.ts

```diff
@@ -1,5 +1,6 @@
 export type Address = `0x${string}`;
 export type Hex = `0x${string}`;
+export type BigintIsh = bigint | string | number;
```

Comment:
moved `BigintIsh` type here since it's usage is spread across a number of files 

 ( originally was declared in `tokenAmount.ts` )